alerts/cluster-network-operator/OVNKubernetesControllerDisconnectedSouthboundDatabase.md

```diff
@@ -0,0 +1,97 @@
+# OVNKubernetesControllerDisconnectedSouthboundDatabase
+
+## Meaning
+
+The `OVNKubernetesControllerDisconnectedSouthboundDatabase` alert is triggered
+when the OVN controller is not connected to OVN southbound database
+for more than 5 minutes.
+
+## Impact
+
+Networking control plane is degraded on the node. Existing workloads
+on the node may continue to have connectivity but any networking configuration
+update will not be applied.
+
+## Diagnosis
+
+### Fix alerts before continuing
+
+Check to ensure the following alerts are not firing and resolved before
+continuing as they may cause this alert to fire:
+
+- [NoRunningOvnMaster](./NoRunningOvnMaster.md)
+
+### OVN-kubernetes master pods
```

Comment:
maybe call that like "Check sbdb".  We need to find sbdb leader and check only its logs, because this instance will accept connections.
We can find sbdb leader with a network-tools command `oc adm must-gather --image=quay.io/openshift/origin-network-tools:latest -- network-tools ovn-get leaders` and then check logs of sbdb on returned pod only instead of all pods? Also, if no leader will be found by this command, this is a good place to create a bug (and hopefully soon we will have a separate alert+runbook for no db leader)
wdyt?

Reply:
I don't think that ovn-controller can only connects to the leader sbdb:
```
for pod in $(oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node -o jsonpath={..metadata.name})
do
  echo "${pod}:"
  oc logs ${pod} -n openshift-ovn-kubernetes -c ovn-controller | grep "ssl.*connected"
done
ovnkube-node-2bmrc:
2022-09-08T09:12:57.096Z|00010|reconnect|INFO|ssl:10.0.128.155:9642: connected
ovnkube-node-68pwf:
2022-09-08T09:07:06.821Z|00024|reconnect|INFO|ssl:10.0.128.155:9642: connected
ovnkube-node-dhnks:
2022-09-08T09:12:54.577Z|00008|reconnect|INFO|ssl:10.0.168.127:9642: connected
ovnkube-node-ppjbq:
2022-09-08T09:07:06.812Z|00024|reconnect|INFO|ssl:10.0.149.143:9642: connected
ovnkube-node-psdrj:
2022-09-08T09:13:00.102Z|00008|reconnect|INFO|ssl:10.0.168.127:9642: connected
ovnkube-node-rp8hv:
2022-09-08T09:07:08.338Z|00024|reconnect|INFO|ssl:10.0.128.155:9642: connected
```
So I think it is still worth to check all of the SBDBs as they can be potentially picked as an endpoint.

Reply:
correct, to avoid scaling issues, 1/3rd of controllers connect to sbdbA 1/3rd to sbdbB and 1/3rd to sbdbC